modules/lambdas/main.tf

```diff
@@ -1,5 +1,6 @@
 # Local Variables
 locals {
+  tags         = merge(var.tags, { Deployment = var.prefix })
```

Comment:
I think you will need to merge develop branch into this branch to pull in the latest changes. 

In the develop branch, it no longer has the local tags value.

Reply:
I changed this back. The original intention was to keep tags self-contained so we wouldn't need to rely on a .tfvars file but it shouldn't matter either way

Reply:
looks good now